test/sql/updates/setup.continuous_aggs.v2.sql

```diff
@@ -437,3 +449,70 @@ BEGIN
     PERFORM alter_job(retention_jobid, scheduled=>false);
   END IF;
 END $$;
+
+-- Test that calling drop chunks on the hypertable does not break the
+-- update process when chunks are marked as dropped rather than
+-- removed. This happens when a continuous aggregate is defined on the
+-- hypertable, so we create a hypertable and a continuous aggregate
+-- here and then drop chunks from the hypertable and make sure that
+-- the update from 1.7 to 2.0 works as expected.
+CREATE TABLE drop_test (
+    time timestamptz,
+    location INT,
+    temperature double PRECISION
+);
+
+SELECT create_hypertable ('drop_test', 'time', chunk_time_interval => interval '1 week');
+
+INSERT INTO drop_test
+SELECT
+    time,
+    (random() * 3 + 1)::int,
```

Comment:
do we set seed somewhere for that? if not maybe it makes sense to do that before this test case

Reply:
Just using this data to generate chunks, so only the number of rows is important.